### DISCUSSION TITLE
vfmt: do not change breaks inside expressions

### DISCUSSION THREAD

**Discussion by UweKrueger** [vlang/v#11739] [Ideas]
While `vfmt` uses a sophisticated algorithm to perform automatic line breaks the decision where to break a complex expression often does not satisfy the expectation of the programmer.
Suggestion: Just use the break positions that are given by the programmer for expressions.

**Comment by medvednikov** [comment]
A note: this is how gofmt works.

**Comment by lilenvy** [comment]
to be fair gofmt does swallow line breaks up to 1, but it does respect single breaks, I find it hard on V to create array literals because of this. I mean it works and I don't pay attention too much, but sometimes I wish it respected _some_  breaks

**Comment by medvednikov** [comment]
@lilenvy it will work just like gofmt
